src/codegeneration/generator/CPSTransformer.js

```diff
@@ -164,7 +165,13 @@ export class CPSTransformer extends TempVarTransformer {
     super(identifierGenerator);
     this.reporter = reporter;
     this.stateAllocator_ = new StateAllocator();
-    this.labelSet_ = Object.create(null);
+
+    // This is currently a Map<string, LabelState> where the key is the
+    // label.name. We should probably change this to Set<LabelState> but that
+    // requires us depending on a real Set which might be too slow in old JS
+    // engines.
```

Comment:
Can you add any specifics here? As in, if we read this some time in the future will we know that its now ok to switch to Set.